src/sysupdate/sysupdated.c

```diff
@@ -816,16 +816,20 @@ static int sysupdate_run_simple(sd_json_variant **ret, Target *t, ...) {
 
         r = sd_json_parse_file(f, "stdout", 0, &v, NULL, NULL);
         if (r < 0)
-                return log_error_errno(r, "Failed to parse JSON: %m");
+                return log_debug_errno(r, "Failed to parse JSON: %m");
```

Comment:
hmm, why this change? this should be logged about, no? and given the flags passed here aren't set to SD_JSON_LOG there would be no logging here at all now?

(actually logging here is all over the place?

Reply:
Otherwise we log twice about the same error. Once here, and once at some of the call sites of `sysupdate_run_simple`. The commit makes sure that all callsites of `sysupdate_run_simple` log now, and sliences `sysupdate_run_simple`'s own logging to avoid double-logging the errors.

Reply:
re: flags, we're not dispatching here, we're parsing. SD_JSON_LOG is a dispatch flag